src/Discord.Net.WebSocket/Entities/Interaction/SocketBaseCommand/SocketApplicationCommand.cs

```diff
@@ -49,7 +49,7 @@ public DateTimeOffset CreatedAt
             => SnowflakeUtils.FromSnowflake(Id);
 
         /// <summary>
-        ///     Returns the guild this command resides in, if this command is a global command then it will return <see langword="null"/>
+        ///     Gets the guild this command resides in, if this command is a global command then it will return <see langword="null"/>
```

Comment:
```suggestion
        ///     Gets the guild this command resides in; if this command is a global command then it will return <see langword="null"/>
```